Pull request for issue #13 (#20)
ices-eg/wg_WGFAST: SONAR-netCDF4/docs/crr341.adoc

==== Platform group

This group contains information about the sonar platform (e.g. ship). The netCDF4 group name is */Platform* and is described in <<platformGroupTable>>. 

> Optionally, /Platform subgroups can be included to store data from individual instruments measuring values that are relative to the platform. If those values are relative to the environment around the platform, they must be stored in the Environment group. 

Reviewer: ```suggestion
Optionally, subgroups of /Platform can be used to store data from individual instruments that provide measurements about the platform (note that if those values are relative to the environment around the platform, they should be stored in the Environment group). 
```

> Each platform can be made up with several instruments belonging to the same category. (For example a platform can associated to several position sensors). Each category is defined as a subgroup of the Platform group and data for each sensor are stored in a subgroup of the sensor category. This structure must be used to store any type of sensor. 

Reviewer: ```suggestion

Each platform can have several instruments belonging to the same category (for example, a platform can have several position sensors). Each category is defined as a subgroup of the Platform group and data for each sensor are stored in a subgroup of the sensor subgroup. This structure can be used to store any type of platform sensor. 
```

> More specifically individual position and (resp.) attitude instruments are defined individually as subgroups of the */Platform/Position* and (resp.) */Platform/Attitude* subgroups. The name of those subgroups must match the names of their serial number or identifier as defined in <<platformGroupTable>>.

Reviewer: ```suggestion
For example, data from individual position and attitude instruments are stored as subgroups of the */Platform/Position* and */Platform/Attitude* subgroups. The names of those subgroups must match the names of the instrument's serial number or identifier as defined in <<platformGroupTable>>.
```

The coordinate system convention, including the definition of pitch and roll, is detailed in <<_coordinate_systems>>.

> Each sensor instance group would also provide a standard place to store unprocessed and unparsed instrument data from multiple GPS sensors, motion reference units, etc. 

Reviewer: ```suggestion

Each sensor instance group would also provide a standard place to store unprocessed and unparsed instrument data from multiple GPS sensors, motion reference units, etc. 
```

The data format is not prescribed, but could be, for example, be NMEA-style text and/or numeric values. Subgroups must have one attribute called “description” that provides a short description of the contents. Other attributes can be added as desired. The variables under the subgroup should have appropriate names and an attribute that gives the units, where appropriate.

> The following table give the suggested convention recommended to name subgroups for different known kind of sensors or datagrams

Reviewer: ```suggestion
<<platformInstrumentsTable>> gives recommended platform sensor subgroup names.
```

> [cols=",,",options="header",]

Reviewer: ```suggestion
.Suggested subgroup names for platform instruments.
[cols=",,",options="header",]
[[platformInstrumentsTable]]
```
